app/repositories.php

```diff
@@ -46,6 +47,8 @@
             return $repo;
         },
 
+        CampaignStatisticsRepository::class => static fn(ContainerInterface $c): CampaignStatisticsRepository => new CampaignStatisticsRepository($c->get(EntityManagerInterface::class)),
```

Comment:
Not a problem to have this but I think it's not necessary - the DI container will do this automatically with reflection if we don't have this line. We just need to list things where the way to instantiate is less obvious.

Reply:
Yep true, removed in 2c8bc50e37e03aa82d2266fff210d0044a5e0942